<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>input</class>
 <widget class="QWidget" name="input">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgba(249, 255, 181, 250);

</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1680</x>
     <y>200</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(20, 204, 20);</string>
   </property>
   <property name="text">
    <string>출발역</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_time">
   <property name="geometry">
    <rect>
     <x>1550</x>
     <y>599</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border: 2px solid rgb(20, 204, 20);</string>
   </property>
   <item>
    <property name="text">
     <string>00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>01</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>02</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>03</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>04</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>05</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>06</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>07</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>08</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>09</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="l_start">
   <property name="geometry">
    <rect>
     <x>1550</x>
     <y>270</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
qproperty-alignment: 'AlignCenter';
border: 2px solid rgb(20, 204, 20);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1600</x>
     <y>550</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string>color: rgb(20, 204, 20);</string>
   </property>
   <property name="text">
    <string>시간</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1680</x>
     <y>330</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string>color: rgb(20, 204, 20);</string>
   </property>
   <property name="text">
    <string>도착역</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_min">
   <property name="geometry">
    <rect>
     <x>1730</x>
     <y>600</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border: 2px solid rgb(20, 204, 20);</string>
   </property>
   <item>
    <property name="text">
     <string>00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>05</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>35</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>45</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>55</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>440</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string>color: rgb(20, 204, 20);</string>
   </property>
   <property name="text">
    <string>요일</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1790</x>
     <y>550</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string>color: rgb(20, 204, 20);</string>
   </property>
   <property name="text">
    <string>분</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="r_mon">
   <property name="geometry">
    <rect>
     <x>1550</x>
     <y>500</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>월</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="r_tue">
   <property name="geometry">
    <rect>
     <x>1600</x>
     <y>500</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>화</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="r_wed">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>500</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>수</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="r_thur">
   <property name="geometry">
    <rect>
     <x>1700</x>
     <y>500</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>목</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="r_sat">
   <property name="geometry">
    <rect>
     <x>1800</x>
     <y>500</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>토</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="r_fri">
   <property name="geometry">
    <rect>
     <x>1750</x>
     <y>500</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>금</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="r_sun">
   <property name="geometry">
    <rect>
     <x>1850</x>
     <y>500</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>일</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="l_end">
   <property name="geometry">
    <rect>
     <x>1550</x>
     <y>382</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
qproperty-alignment: 'AlignCenter';
border: 2px solid rgb(20, 204, 20);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1501</width>
     <height>1061</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_put">
   <property name="geometry">
    <rect>
     <x>1530</x>
     <y>700</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(20, 204, 20);
	background-color: white;
	border: 4px solid rgb(20, 204, 20);
	border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>노선도 검색</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_sr">
   <property name="geometry">
    <rect>
     <x>1730</x>
     <y>700</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(20, 204, 20);
	background-color: white;
	border: 4px solid rgb(20, 204, 20);
	border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>혼잡도 검색</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_re">
   <property name="geometry">
    <rect>
     <x>1630</x>
     <y>930</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>오뮤 다예쁨체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(251, 86, 7);
	background-color: white;
	border: 4px solid rgb(251, 86, 7);
	border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>돌아가기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
